dvd/dvd_dump.ixx

```diff
@@ -826,9 +826,18 @@ export bool redumper_dump_dvd(Context &ctx, const Options &options, DumpMode dum
                 sectors_to_read = std::min(sectors_to_read, xbox_lock_sector - s);
             else if(s == xbox_lock_sector)
             {
+                status = cmd_read_capacity(*ctx.sptd, sector_last, block_length, false, 0, false);
```

Comment:
Not sure if this is necessary, shouldn't it have the value from before?

Reply:
You're right, fixed.